### DISCUSSION TITLE
Internationalization

### DISCUSSION THREAD

**Discussion by clement-escolano** [plausible/analytics#465] [Anything Else]
Hello,
Thank you for Plausible, this is a really nice tool (and more beautiful coming from Matomo 😄).
Do you have plans to integration i18n into Plausible? As I understand, only English is supported.
Thanks

**Comment by metmarkosaric** [comment]
Thanks @clement-escolano! Could be nice for sure as currently only English is supported. There's also #354 for the same request. This work hasn't been planned yet. You can see what's on [our current roadmap](https://github.com/plausible/analytics/projects/1). But as usual, we're open to the community to contribute and help lead the prioritization of the roadmap for us.

**Comment by clement-escolano** [comment]
Sorry, I search for similar issues but did not found this one. Is there a way to close this issue as it is a duplicate?
Thank you for your answer. I don't have time to work on it right now but I am very interested in plausible supporting other languages so I may work on it some day :-)
Cheers

**Reply by metmarkosaric** [reply]
No problem. I'll lock this thread. 

#354
